documentation/modules/post/networking/gather/enum_cisco.md

```diff
@@ -48,7 +48,7 @@ resource (cisco.rb)> run
 [*] Command shell session 1 opened (111.111.1.111:40721 -> 222.222.2.222:22) at 2019-07-20 16:29:05 -0400
 [*] Scanned 1 of 1 hosts (100% complete)
 [*] Auxiliary module execution completed
-resource (cisco.rb)> use post/cisco/gather/enum_cisco
+resource (cisco.rb)> use post/networking/gather/enum_cisco
 resource (cisco.rb)> set session 1
```

Comment:
Is this resource prompt needed? Or should this not be msf's default prompt followed by the module specific prompt? I don't see any references to loading up a resource file here so I'm confused how and why the documentation shows the prompt as `resource (cisco.rb)` as you don't seem to be loading a resource. This applies to all prompts in this documentation.